src/opm/input/eclipse/EclipseState/SimulationConfig/BCConfig.cpp

```diff
@@ -23,7 +23,7 @@
 #include <opm/input/eclipse/Parser/ParserKeywords/B.hpp>
 #include <opm/input/eclipse/EclipseState/SimulationConfig/BCConfig.hpp>
 #include <opm/common/utility/OpmInputError.hpp>
-
+#include <opm/input/eclipse/Schedule/BCProp.hpp>
```

Comment:
This change seems unnecessary, I do not see any usage of BCProp in the file.

Reply:
fixed.